tests/jest.config.dom.js

```diff
@@ -11,4 +11,10 @@ module.exports = {
     '/__node_tests__/',
   ],
   testEnvironment: 'jest-environment-jsdom',
+  transform: {
+    '^.+\\.(js|jsx|ts|tsx)$': path.join(
+      __dirname,
+      '../node_modules/kcd-scripts/dist/config/babel-transform',
+    ),
+  },
```

Comment:
Could you make a PR to kcd-scripts so we don't have to do this?

https://github.com/kentcdodds/kcd-scripts/blob/50170f0b090593c39f30edc997f62af1c00ecfe1/src/config/jest.config.js#L50